src/player.c

```diff
@@ -534,7 +534,7 @@ static void PlayerUsePortal(Entity* this) {
     }
 }
 
-static void PortalJumpOnUpdate(Entity* this) {
```

Comment:
Why is static removed?

Reply:
I needed to remove static here (and for a few other functions), because they are now pointed to from assembly (in this case from `gUnk_0811BA9C` in ` data_0811B9E0.s`.
Static functions don't emit labels, so I needed to remove the static for now.
Once `gUnk_0811BA9C` is defined in the `player.c` file (and the same for other pointer arrays), static can be added again. (Or is there a way to temporarily generate labels for static functions as well? This would also make some of my tools better as static functions do not appear in `tmc.map`)

Reply:
Gotcha, sorry. Thought that was one I had dumped by hand already. LGTM!